src/api/spec/models/package_spec.rb

```diff
@@ -230,6 +230,19 @@
     end
   end
 
+  describe '#serviceinfo' do
+    let(:url) { "#{CONFIG['source_url']}/source/#{package_with_service.project}/#{package_with_service.name}" }
+    let(:no_serviceinfo) do
+      '<directory name="package_with_service" rev="1" vrev="1" srcmd5="cf9c84e27a27dfc3e289f74fb096b42a">
```

Comment:
you can do multiline strings like this? And why is everyone else doing the <<~XML dance? I feel underinformed!! :)

Reply:
We just C&P from the spec for service_error...

Reply:
I like `file_fixture` best...

Reply:
I don't. They hide what we test and make the test harder to read. For oneliner strings (shorter strings in general) I would prefer the heredoc syntax

Reply:
If we achieve an agreement on that, I'd do it in a new PR and refactor the whole file.
I have no strong opinion, but I think I'd prefer `file_fixture`... :thinking:

Reply:
I'm with @bgeuken - the extra file makes it harder

Reply:
> If we achieve an agreement on that

We don't need to. This was just a conversation :smile: